website/docs/upgrade-guides.md

```diff
@@ -5,6 +5,106 @@ sidebar_label: Upgrade Guides
+{ label: 'Android', value: 'android', }
+]}>
+<TabItem value="ios">
+
+#### Dependency Version Alignment
+
+The only necessary steps to use Portals version 0.7.0 should be to update dependencies. IonicPortals for iOS version 0.7.0 is compatible with Live Updates 0.2.2 and Official Capacitor Plugins over version 4.0.
+
+</TabItem>
+
+<TabItem value="android">
+
+#### Dependency Version Alignment
+
+IonicPortals for Android version 0.7.0 is compatible with the following dependency versions. Update as needed:
+
+<CodeBlock className="language-groovy" title="build.gradle">
+{
+`
+dependencies {
+    implementation 'io.ionic:portals:0.7.0'
+    implementation 'io.ionic:liveupdates:0.2.0'
+    implementation 'com.capacitorjs:core:4.5.0'
+    // Any Official Capacitor Plugins over version 4.0
+}`.trim()
+}
+</CodeBlock>
+
+#### Gradle Compatibility
+
+Make sure your Android project is using Gradle 7.2.2 or higher. If your project is using an older Gradle version, Android Studio may prompt you to use the [Android Gradle Plugin Upgrade Assistant](https://developer.android.com/studio/build/agp-upgrade-assistant) to update your project. You may use this tool.
+
+#### CompileSdk and TargetSdk Versions
+
+Projects should be updated to compile and target Android SDK version 32 or higher (Android 12).
+
+#### Add the android:exported tag
+
+In your `AndroidManifest.xml` file you'll need to add the `android:exported` tag to the Activity. Launching activities should have the value set to `true`. For more details see [android:exported](https://developer.android.com/guide/topics/manifest/activity-element#exported) in the Android docs.
+
+#### Resolving Dependency Issues
+
+You may encounter further dependency version conflicts when building your app, such as the following:
+
+```
+Execution failed for task ':app:checkDebugDuplicateClasses'.
+> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
+   > Duplicate class androidx.lifecycle.ViewModelLazy found in modules jetified-lifecycle-viewmodel-ktx-2.3.1-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1) and lifecycle-viewmodel-2.5.1-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.1)
+     Duplicate class androidx.lifecycle.ViewTreeViewModelKt found in modules jetified-lifecycle-viewmodel-ktx-2.3.1-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1) and lifecycle-viewmodel-2.5.1-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.1)
+```
+
+To resolve this, update any conflicting dependencies in your project. If you require newer dependency versions than those used in Portals that are conflicting, use Gradle to [override transitive dependencies](https://stackoverflow.com/questions/30728533/gradle-override-transitive-dependency-by-version-classifier).
+
+</TabItem>
+
+</Tabs>
+
+### InitialContext Update
+
+A default InitialContext is now always passed to the web content such that the Portal name will always be accessible.
```

Comment:
Little nitpick, we should probably specify it will always be accessible when running in a Portals context.